src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,75 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 14px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    min-height: 20vh;
+}
+
+.mdhui-chat p,
+.mdhui-chat ul {
+    margin: .25rem 0;
+    word-break: break-word;
+    color: var(--mdhui-text-color-1);
+}
+
+.mdhui-chat input,
+.mdhui-chat button {
+    min-height: var(--mdhui-touch);
+    min-width: var(--mdhui-touch);
+    border: 0;
+    border-radius: var(--mdhui-card-border-radius);
+}
+
+.mdhui-chat .send-button {
+    position: absolute;
+    right: 4px;
+    cursor: pointer;
+    text-align: center;
+}
+
+.mdhui-chat input {
+    flex: 1;
+    font-size: 1em;
+    color: var(--mdhui-text-color-1);
+    padding: var(--mdhui-padding-md) var(--mdhui-touch) var(--mdhui-padding-md) var(--mdhui-padding-md);
+    box-sizing: border-box;
+    outline-offset: -1px;
+}
+
+.mdhui-chat .input-group {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    background: #fff;
+    border-top: 1px solid var(--mdhui-border-color-1);
+}
+
+.mdhui-chat .mdhui-chat-message,
+.mdhui-chat .message-loading {
+    padding: var(--mdhui-padding-xs) var(--mdhui-padding-xs) var(--mdhui-padding-xs) var(--mdhui-padding-lg);
+    position: relative;
+    border-bottom: 1px solid var(--mdhui-border-color-1);
+}
+
+.mdhui-chat .mdhui-chat-message.sent {
+    background: var(--mdhui-background-color-1);
+}
+
+.mdhui-chat .mdhui-chat-message .svg-inline--fa,
+.mdhui-chat .message-loading .svg-inline--fa {
+    color: var(--mdhui-text-color-3);
+    margin-right: .25rem;
```

Comment:
I would flex in the icon or position it absolutely so it's not just inline with the text, e.g.
<img width="404" alt="Screenshot 2024-08-15 at 2 56 55 PM" src="https://github.com/user-attachments/assets/1be2b367-1e0a-43ed-aa70-5e4192d27725">

Reply:
Actually, I would just use "Action" component for the messages.  That component already supports icons on the left etc. and is visually consistent - and, despite it's name, if you do not supply an onclick handler it will not be clickable.  Then you do not need to style the messages at all - you can just put the message itself in the subtitle or children

Reply:
Argh the first time it was positioned absolutely and you said `I would display:flex this with the message and not position it absolutely actually` which I think I also misinterpreted as meaning displaying the icon inline. Will try with flex.